packages/react-native-editor/package.json

```diff
@@ -80,7 +80,7 @@
 		"start": "echo \"\\x1b[33mThe start command is not available in this project. It is strongly recommended to use \\x1b[1:33mstart:reset\\x1b[0m\\x1b[33m to perform some cleanup when starting the metro bundler.\nOr you may use \\x1b[1:33mstart:quick\\x1b[0m\\x1b[33m for a quicker startup, but this may lead to unexpected javascript errors when running the app.\\x1b[0m\"",
 		"start:debug": "node --inspect-brk node_modules/.bin/react-native start",
 		"start:reset": "npm run clean:runtime && npm run start:quick -- --reset-cache",
-		"start:quick": "react-native start",
+		"start:quick": "react-native start --experimental-debugger",
```

Comment:
Do you think we should enable this globally or let contributors pass this as desired via `npm run start:quick -- --experimental-debugger`?

Reply:
Good question, I think it's handy to have it and you just decide if you want to open it or not.